apps/emqx_authn/src/emqx_authn_api.erl

```diff
@@ -919,6 +920,14 @@ serialize_error({unknown_authn_type, Type}) ->
 serialize_error({bad_authenticator_config, Reason}) ->
     {400, #{code => <<"BAD_REQUEST">>,
             message => binfmt("Bad authenticator config ~p", [Reason])}};
+serialize_error(#{reason := {Mod, ValidationErrors}, type := Type} = Error) ->
+    ?SLOG(error, Error),
```

Comment:
Do we intentionally log only this concrete error?

Reply:
sorry for this error, I will remove this, add a test case to cover this.